scripts/install_test_certs.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+mkdir -p /tmp/test-ca
+
+wget https://github.com/OpenVPN/easy-rsa/releases/download/3.0.1/EasyRSA-3.0.1.tgz -O /tmp/EasyRSA.tgz
```

Comment:
Would it be okay to vendor these deps? Pulling in deps during build is usually not good because it adds another layer that can break and reproducible builds are cleaner IMO